### ISSUE TITLE
Gmoccapy - clicking in preview makes bug

### ISSUE THREAD

**Issue by zz912** [LinuxCNC/linuxcnc#2631]
Clicking in preview makes bug:
![Clicking_in_preview](https://github.com/LinuxCNC/linuxcnc/assets/96618597/1deb4424-abf9-4aec-8422-afaa042d6ccf)
```
[Gmoccapy][ERROR]  Found an error!
The following information may be useful in troubleshooting:

Traceback (most recent call last):
  File "/home/linaplast-zz/linuxcnc/linuxcnc-2.9-test/lib/python/rs274/glcanon.py", line 410, in inner
    return f(self, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/linaplast-zz/linuxcnc/linuxcnc-2.9-test/lib/python/gremlin.py", line 496, in select_fire
    self.select(x, y)
  File "/home/linaplast-zz/linuxcnc/linuxcnc-2.9-test/lib/python/rs274/glcanon.py", line 595, in select
    buffer = glRenderMode(GL_RENDER)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/OpenGL/GL/pointers.py", line 260, in glRenderMode
    raise error.GLError(
OpenGL.error.GLError: GLError(
	err = GL_STACK_OVERFLOW,
	description = b'stack overflow',
	result = 'glSelectBuffer too small to hold selection results'
)
 (gmoccapy:62)

```

**Comment by c-morley** [comment]
This is lower in the code then Gmoccapy - affects Qtvcp and Gladevcp haven't heard about an AXIS report yet. I may have more details in the qtvcp section of forum...

**Comment by c-morley** [comment]
I can't find it. IIRC they changed something in the underlying openGL python library (something we can't do of course) but it might give us a clue. I'll post if I find it.

**Comment by gmoccapy** [comment]
I can reproduce this. affects all UIs that use gremlin.

**Comment by gmoccapy** [comment]
Fixed 07.10.23 in 2.9
